google/cloud/forseti/enforcer/gce_firewall_enforcer.py

```diff
@@ -185,6 +185,16 @@ def filter_rules_by_network(rules, network):
     return filtered_rules
 
 
+def _rule_update_can_patch(rule_from, rule_to):
```

Comment:
It would be nice to see unit tests for this.

Reply:
captured in test_apply_firewall_denied_to_allowed